Content.Shared/Damage/DamageType.cs

```diff
@@ -29,7 +30,7 @@ public static DamageClass ToClass(this DamageType type)
             return DamageSystem.TypeToClass[type];
         }
 
-        public static Dictionary<DamageType, T> ToNewDictionary<T>()
+        public static Dictionary<DamageType, T?> ToNewDictionary<T>()
```

Comment:
Paging @DrSmugleaf

Reply:
`where T : struct` instead

Reply:
Changed to `public static Dictionary<DamageType, T> ToNewDictionary<T>() where T : struct` as Smug suggested.